docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,931 @@
+[role="xpack"]
+[[ootb-ml-jobs]]
+== Prebuilt {anomaly-jobs}
+
+{anomaly-jobs-cap} contain the configuration information and metadata necessary 
+to perform an analytics task. {kib} can recognize certain types of data and 
+provide specialized wizards for that context. This page lists the available 
+prebuilt {anomaly-jobs} that you can use via {kib}.
+
+
+[float]
+[[ootb-ml-jobs-apache]]
+=== Apache
+
+// tag::apache-jobs[]
+These {anomaly-job} wizards appear in {kib} if you use 
+{filebeat-ref}/index.html[{filebeat}] to ship access logs from your 
+https://httpd.apache.org/[Apache] HTTP servers to {es} and store it using fields 
+and datatypes from the Elastic Common Schema (ECS).
+
+low_request_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `apache.access`.
+* Models the event rate of HTTP requests. 
+* Detects unusually low counts of HTTP requests compared to the previous event 
+  rate (using the <<ml-count,`low_count` function>>).
+
+////
+Bucket span: 15m.
+
+Function: `low_count`.
+////
+
+source_ip_request_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `apache.access`.
+* Models the event rate of HTTP requests by source IP.
+* Detects source IPs with unusually high request rates in the HTTP access log 
+  compared to the previous rate (using the <<ml-count,`high_count` function>>).
+
+////
+Influencers:
+
+* `source.address`
+
+Bucket span: 1h.
+
+Function: `high_count`.
+////
+
+source_ip_url_count_ecs::
+
+* For HTTP web access logs where `event.dataset` is `apache.access`.
+* Models the event rate of HTTP requests by source IP.
+* Detects source IPs with unusually high distinct count of URLs in the HTTP 
+access log (using the <<ml-distinct-count,`high_distinct_count` function>>).
+
+////
+Influencers:
+
+* `source.address`
+
+Bucket span: 1h.
+
+Function: `high_distinct_count`.
+////
+
+status_code_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `apache.access`.
+* Models the occurrences of HTTP response status codes (`partition_field_name` 
+  is `http.response.status_code`).
+* Detects unusual status code rates in the HTTP access log compared to previous 
+  rates (using the <<ml-count,`count` function>>).
+
+////
+Influencers:
+
+* `http.response.status_code` 
+* `source.address`
+
+Bucket span: 15m.
+
+Function: `count`.
+////
+
+visitor_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `apache.access`.
+* Models visitor rates.
+* Detects unusual visitor rates in the HTTP access log ompared to previous 
+  rates (using the <<ml-nonzero-count,`non_zero_count` function>>).
+
+////
+Bucket span: 15m.
+
+Function: `non_zero_count`.
+////
+// end::apache-jobs[]
+
+[float]
+[[ootb-ml-jobs-apm]]
+=== APM
+
+// tag::apm-jobs[]
+abnormal_span_durations_jsbase::
+abnormal_span_durations_nodejs::
+
+* For APM data where `agent.name` is `js-base` or `nodejs`.
```

Comment:
@blaklaybul I've heard `rum-js` replaces `js-base`. Do our jobs pick those up too?:
```suggestion
* For APM data where `agent.name` is `js-base`, `rum-js`, or `nodejs`.
```

Reply:
I was wrong. It looks like `agent.name` hasn't been changed to `rum-js` yet, but it will be soon. Here's the open PR https://github.com/elastic/apm-agent-rum-js/pull/548.

Reply:
The PR I linked above is the last change required to rename the Agent (also see https://github.com/elastic/apm-agent-rum-js/issues/108). Not sure if it's too late, but it might make more sense to name this `abnormal_span_durations_rumjs` instead of `abnormal_span_durations_jsbase`. A similar rename for the other anomaly detection jobs to `*_rumjs` might be considered.

Reply:
Per a quick chat with @blaklaybul, we can/should update the name when APM makes the change